mobile/src/bundles/common/components/modals/active-modal/active-modal.tsx

```diff
@@ -16,7 +16,7 @@ type Properties = {
     title: string;
     visible: boolean;
     acceptText?: string;
-    children: React.ReactNode;
+    children?: React.ReactNode;
```

Comment:
Modal can be empty?

Reply:
Title is required, so if we pass title it will look like on the screenshot.